hack/.spelling_failures

```diff
@@ -7,6 +7,7 @@ docs/man
 docs/user-guide
 generated.pb.go
 generated.proto
+go.sum
```

Comment:
should I also add `go.mod` here?  Its not currently failing the check but it easily could due to github handles.

Reply:
sounds like a good idea to me

Reply:
added.